test/src/unittests/stats/test_histogram.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2006-2016  Music Technology Group - Universitat Pompeu Fabra
+#
+# This file is part of Essentia
+#
+# Essentia is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation (FSF), either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the Affero GNU General Public License
+# version 3 along with this program. If not, see http://www.gnu.org/licenses/
+
+
+from essentia_test import *
+
+testdir = join(filedir(), 'histogram')
+
+class TestHistogram(TestCase):
+
+  def testZero(self):
+    self.assertEqualVector(Histogram()(zeros(1000)), [[1000], [0.5]])
```

Comment:
For better readability, set the default parameters you use explicitly in the test code.

Reply:
I was referring to the parameters (min/max frequency, number of bins, etc.), not the input and output.